oauth-client-credentials-with-scope/deploy-oauth-client-credentials-with-scope.sh

```diff
@@ -0,0 +1,110 @@
+apigeecli products create --name oauth-client-credentials-product-with-write-scope --displayname "oauth-client-credentials-product-with-write-scope" --opgrp ./oauth-client-credentials-product-ops.json --envs "$APIGEE_ENV" --approval auto --quota 10 --interval 1 --unit minute --org "$PROJECT" --token "$TOKEN" --scopes "read" --scopes "write"
+
+echo "Creating Developer"
+apigeecli developers create --user testuser --email oauth-client-credentials-with-scope_apigeesamples@acme.com --first Test --last User --org "$PROJECT" --token "$TOKEN"
+
+echo "Creating Developer Apps"
+apigeecli apps create --name $APP_NAME_READ_SCOPE --email oauth-client-credentials-with-scope_apigeesamples@acme.com --prods oauth-client-credentials-product-with-read-scope --org "$PROJECT" --token "$TOKEN" --disable-check
+apigeecli apps create --name $APP_NAME_WRITE_SCOPE --email oauth-client-credentials-with-scope_apigeesamples@acme.com --prods oauth-client-credentials-product-with-write-scope --org "$PROJECT" --token "$TOKEN" --disable-check
+
+APP_READ_SCOPE_CLIENT_ID=$(apigeecli apps get --name $APP_NAME_READ_SCOPE --org "$PROJECT" --token "$TOKEN" --disable-check | jq ."[0].credentials[0].consumerKey" -r)
+export APP_READ_SCOPE_CLIENT_ID
+APP_READ_SCOPE_CLIENT_SECRET=$(apigeecli apps get --name $APP_NAME_READ_SCOPE --org "$PROJECT" --token "$TOKEN" --disable-check | jq ."[0].credentials[0].consumerSecret" -r)
+export APP_READ_SCOPE_CLIENT_SECRET
+
+APP_WRITE_SCOPE_CLIENT_ID=$(apigeecli apps get --name $APP_NAME_WRITE_SCOPE --org "$PROJECT" --token "$TOKEN" --disable-check | jq ."[0].credentials[0].consumerKey" -r)
+export APP_WRITE_SCOPE_CLIENT_ID
+APP_WRITE_SCOPE_CLIENT_SECRET=$(apigeecli apps get --name $APP_NAME_WRITE_SCOPE --org "$PROJECT" --token "$TOKEN" --disable-check | jq ."[0].credentials[0].consumerSecret" -r)
+export APP_WRITE_SCOPE_CLIENT_SECRET
+
+# var is expected by integration test (apickli)
+export PROXY_URL="$APIGEE_HOST/v1/samples/oauth-client-credentials-with-scope"
+
+# integration tests
+npm run test
+
+echo " "
+echo "All the Apigee artifacts are successfully deployed!"
+echo " "
+echo "Your Proxy URL is: https://$PROXY_URL"
+echo "Your app client id with read scope is: $APP_READ_SCOPE_CLIENT_ID"
+echo "Your app client secret with read scope is: $APP_READ_SCOPE_CLIENT_SECRET"
+echo " "
+echo "Your app client id with write scope is: $APP_WRITE_SCOPE_CLIENT_ID"
+echo "Your app client secret with write scope is: $APP_WRITE_SCOPE_CLIENT_SECRET"
+echo " "
+echo "-----------------------------"
+echo " "
+echo "Generating a read scope token using $APP_NAME_READ_SCOPE credentials"
+READ_TOKEN=$(curl -s POST https://"$PROXY_URL"/token -u "$APP_READ_SCOPE_CLIENT_ID":"$APP_READ_SCOPE_CLIENT_SECRET" -d "grant_type=client_credentials&scope=read" | jq ."access_token" -r )export READ_TOKEN
```

Comment:
Outputs:

`Generating a read scope token using oauth-client-credentials-app-with-read-scope credentials
./deploy-oauth-client-credentials-with-scope.sh: line 92: READ_TOKEN: command not found`

Missing line break before `export READ_TOKEN`?